MyCapytain/resources/collections/capitains.py

```diff
@@ -0,0 +1,174 @@
+from MyCapytain.resources.prototypes.metadata import Collection, ResourceCollection
+from MyCapytain.resources.collections.cts import _parse_structured_metadata, XmlCtsCitation
+from MyCapytain.common.utils.xml import xmlparser
+from MyCapytain.common.constants import XPATH_NAMESPACES, Mimetypes, RDF_NAMESPACES
+from MyCapytain.common.reference._capitains_cts import Citation as CitationPrototype
+from MyCapytain.resources.prototypes.capitains import collection as capitains
+from rdflib.namespace import DCTERMS, DC
+
+XPATH_NAMESPACES.update({'dc': "http://purl.org/dc/elements/1.1/", 'dct': 'http://purl.org/dc/terms/'})
+
+
+def _xpathDict(xml, xpath, cls, parent, **kwargs):
+    """ Returns a default Dict given certain information
+
+    :param xml: An xml tree
+    :type xml: etree
+    :param xpath: XPath to find children
+    :type xpath: str
+    :param cls: Class identifying children
+    :type cls: inventory.Resource
+    :param parent: Parent of object
+    :type parent: CtsCollection
+    :rtype: collections.defaultdict.<basestring, inventory.Resource>
+    :returns: Dictionary of children
+    """
+    children = []
+    for child in xml.xpath(xpath, namespaces=XPATH_NAMESPACES):
+        children.append(cls.parse(
+            resource=child,
+            parent=parent,
+            **kwargs
+        ))
+    return children
+
+
+class XmlCapitainsReadableMetadata(capitains.CapitainsReadableMetadata):
+    """ Represents a general Capitains CtsTextMetadata
+
+    """
+    DEFAULT_EXPORT = Mimetypes.PYTHON.ETREE
+    # We may want to update this at some point to XmlCapitainsCitation.
+    CLASS_CITATION = XmlCtsCitation
+
+    @staticmethod
+    def __findCitations(obj, xml, xpath="ti:citation"):
+        """ Find citation in current xml. Used as a loop for xmlparser()
+
+        :param xml: Xml resource to be parsed
+        :param xpath: Xpath to use to retrieve the xml node
+        """
+
+    @classmethod
+    def parse_metadata(cls, obj, xml):
+        """ Parse a resource to feed the object
+
+        :param obj: Obj to set metadata of
+        :type obj: XmlCapitainsTextMetadata
+        :param xml: An xml representation object
+        :type xml: lxml.etree._Element
+        """
+
+        for child in xml.getchildren():
+            if child.tag.startswith('{' + XPATH_NAMESPACES['dc']):
+                lg = child.get("{http://www.w3.org/XML/1998/namespace}lang")
+                obj.metadata.add(DC.term(child.tag.replace('{' + XPATH_NAMESPACES['dc'] + '}', '')),
+                                 child.text, lg)
```

Comment:
If we move away from `Graph`, we can use the node content and not the text only, so we can keep subnodes here.